src/DotVVM.Framework/ViewModel/Serialization/DotvvmDictionaryConverter.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
Please add unit/integration tests on dictionary serialization - I know that there is a UI test, but it cannot check the exact serialization format. We should already have some serialization tests, so this should be another case.